ship.config.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
     packagesToBump: [],
     packagesToPublish: [
       'packages/js-horizontal-slider',
```

Comment:
these packages get migrated in a different PR once this is approved?

Reply:
The migration is already done on an other branch: https://github.com/algolia/ui-components/tree/feat/vdom-pt2 but we will open the second PR once we get some feedbacks on this implementation :D